src/theme.ts

```diff
@@ -158,15 +176,7 @@ export default createTheme({
   },
   navHeight: 100,
   brandSidebarWidth: 250,
-  mobileBrandSidebarWidth,
+  tabletBrandSidebarWidth,
   backgroundColor: '#f4f4f6',
-  breakpoints: {
-    values: {
-      xs: 0,
-      sm: 375,
-      md: 768,
-      lg: 1280,
```

Comment:
May want to consider making 1280 smaller, to handle some landscape tablets. 
Also, I suppose we can remove `xl` if it isn't used anywhere.

Reply:
updated to 1024 and removed xl